internal/concatjs/concatjs.go

```diff
@@ -58,6 +58,15 @@ func ServeConcatenatedJS(manifestPath, root string, fs FileSystem) http.Handler
 			w.Header().Set("Content-Encoding", "gzip")
 		}
 
+		// Write out scripts
+		for _, s := range scripts {
+			fmt.Fprint(writer, s)
+			// Ensure scripts are separated by a newline
+			if !strings.HasSuffix(s, "\n") {
```

Comment:
why bother checking for the suffix on the string, maybe always add the newline?

Reply:
Sure. May as well.